airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/test/util/DestinationCleaner.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.test.util
+
+fun interface DestinationCleaner {
```

Comment:
Dq, why is the cleaner its own thing and not an entry point into the process?

Reply:
like why not have a `cleanup` method on DestinationProcess? I think they're pretty separate concerns - the dest process is just the connector (i.e. it supports spec/check/write)

but the cleaner is a weird out-of-connector thing. @stephane-airbyte 's recent Snowflake PR actually has a good example of what I want this class to do - https://github.com/airbytehq/airbyte/pull/45370/files#diff-c86b086ef3cc7bf564aedd699868dcc38661d51b3cc1bf34616720971866c6f6

(i.e. `show schemas` -> `drop schema`/etc)